bin/test-lxd-devlxd-vm

```diff
@@ -91,7 +91,8 @@ lxc exec v1 -- curl -s --unix-socket /dev/lxd/sock http://custom.socket/1.0/devi
 lxc exec v1 -- curl -s --unix-socket /dev/lxd/sock http://custom.socket/1.0/config | jq
 lxc exec v1 -- curl -s --unix-socket /dev/lxd/sock http://custom.socket/1.0/meta-data | grep -q '#cloud-config'
 
-lxc restart -f v1
+# Use --timeout instead of --force otherwise the filesystem will get corrupted.
+lxc restart v1 --timeout=60
```

Comment:
Can you add this:

```
lxc restart v1 --timeout=60 || lxc restart v1 -f
```

So it always restarts, but gives it a good chance to cleanly restart.